src/list-sorting/index.js

```diff
@@ -1,3 +1,14 @@
-function listSorting(needle, haystack) {}
+function listSorting(needle, haystack) {
+	if (Array.isArray(haystack[0])) {
+		for (let i = haystack.length-1; i >= 0; i--) {
```

Comment:
Doesn't always have to be `i`, can be `row`

Reply:
💯  adding that to my memory bank. Thanks